### ISSUE TITLE
Sofar solar (Alias zucchetti) 1100tl-3300tl G1 rs485 modbus protocol implementation 

### ISSUE THREAD

**Issue by guadax** [jeanmarc77/123solar#66]
Hi, just a question: protocol php file for the above inverter is almost completed (when completed i can post it). The only issues i have is on the variable

$KWHT is the total counter (a Wh precision is needed)

well... modbus register give me only total produce KWH precision (not watt):

![image](https://github.com/jeanmarc77/123solar/assets/66492267/6e8b6b97-aa09-4ece-899a-7faba0945f6b)

without watt/hours precision the main production page is empty (maybe i'm wrong).

Any suggestions?

**Comment by jeanmarc77** [comment]
Hi, 
Yes it is mandatory. Maybe you can also use a daily counter more precise ?
If think mi600-webif.php use some sort of calculation to get a wh precision

**Comment by guadax** [comment]
all the data available below.... so i've to replace the total production(Produzione cumulativa (kw)) with daily production(Produzione giorno (kw))?

![image](https://github.com/jeanmarc77/123solar/assets/66492267/1fa3a70d-d5fd-4cb7-91f4-21638b75f4b3)

**Comment by jeanmarc77** [comment]
i don't think so, the daily counter don't seem to have a Wh precision

**Comment by guadax** [comment]
Seems to work with daily production
![image](https://github.com/jeanmarc77/123solar/assets/66492267/c3d25d58-9672-438d-8d73-2775fc748a46)

**Comment by jeanmarc77** [comment]
Yes i think so, avg power need to have Wh precison or you'll get bumps in the chart

**Comment by guadax** [comment]
no way: adding watt/hour precision manually didn't solve the problem.... so is elsewhere.... could you tell me the php file with implements the avg plotting function?

**Comment by guadax** [comment]
added random last number (watt precision) not equal to zero
![image](https://github.com/jeanmarc77/123solar/assets/66492267/047480a4-2912-44cf-a905-16e811ccb72e)


![image](https://github.com/jeanmarc77/123solar/assets/66492267/d6dba182-f6c9-4e58-9e56-4198d191578f)
